src/libraries/System.Net.Requests/src/System/Net/FileWebRequest.cs

```diff
@@ -25,7 +25,7 @@ public class FileWebRequest : WebRequest, ISerializable
         private bool _syncHint;
         private int _aborted;
 
-#pragma warning disable SYSLIB0014
+#pragma warning disable SYSLIB0014 // WebRequest, HttpWebRequest, ServicePoint, and WebClient are obsolete. Use HttpClient instead.
```

Comment:
Should we just suppress SYSLIB0014 in System.Net.Requests at the project level?  This assembly is entirely about providing these types.  Same for System.Net.ServicePoint.

Reply:
I'm good with that; will do.